src/rtp.cpp

```diff
@@ -138,6 +138,22 @@ void RtpExtensionHeader::writeCurrentVideoOrientation(size_t offset, uint8_t id,
 	buf[1] = value;
 }
 
+void RtpExtensionHeader::writeMid(size_t offset, uint8_t id, string mid) {
```

Comment:
The `writeX` methods write the same kind of SDES headers and could be internally implemented with a common `writeSdesHeader(size_t offset, uint8_t id, const byte *value, size_t size)`.

Reply:
@paullouisageneau What do you think off adding a `writeOneByteHeader` API? I will not remove `writeCurrentVideoOrientation` so we don't break the public API.

I don't believe sharing code with SDES is worthwhile here. The format being the same is coincidental so it feels wrong to conflate them? As a reader I would find that confusing

Reply:
It looks good, thanks! You could keep specialized methods `writeMid(size_t offset, uint8_t id, string mid)` and `writeRid(size_t offset, uint8_t id, string rid)` calling `writeOneByteHeader()` internally like `writeCurrentVideoOrientation()`.

Reply:
Good point, keep it like this!